cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/NoConfidence.hs

```diff
@@ -0,0 +1,263 @@
+      , "--verification-key-file", committeeVkey1Fp
+      ]
+
+  CommitteeColdKeyHash comKeyHash1 <-
+    H.evalEither
+      $ deserialiseFromRawBytesHex (AsHash AsCommitteeColdKey)
+      $ BSC.pack comKeyHash1Str
+
+  let comKeyCred1 = L.KeyHashObj comKeyHash1
+      committeeThreshold = unsafeBoundedRational 0.5
+      committee = L.Committee (Map.fromList [(comKeyCred1, EpochNo 100)]) committeeThreshold
+
+  alonzoGenesis <- evalEither $ first prettyError defaultAlonzoGenesis
+  (startTime, shelleyGenesis') <- getDefaultShelleyGenesis fastTestnetOptions
+  let conwayGenesisWithCommittee =
+        defaultConwayGenesis { L.cgCommittee = committee }
+
+  TestnetRuntime
+    { testnetMagic
+    , poolNodes
+    , wallets=wallet0:_wallet1:_
+    , configurationFile
+    } <- cardanoTestnet
+           fastTestnetOptions
+           conf startTime shelleyGenesis'
+           alonzoGenesis conwayGenesisWithCommittee
+
+  poolNode1 <- H.headM poolNodes
+  poolSprocket1 <- H.noteShow $ nodeSprocket $ poolRuntime poolNode1
+  execConfig <- H.mkExecConfig tempBaseAbsPath poolSprocket1 testnetMagic
+
+  let socketName' = IO.sprocketName poolSprocket1
+      socketBase = IO.sprocketBase poolSprocket1 -- /tmp
+      socketPath = socketBase </> socketName'
+
+  _epochStateView <- getEpochStateView (File configurationFile) (File socketPath)
+
+  H.note_ $ "Sprocket: " <> show poolSprocket1
+  H.note_ $ "Abs path: " <> tempAbsBasePath'
+  H.note_ $ "Socketpath: " <> socketPath
+
+  res <- findCondition (committeeIsPresent False) configurationFile socketPath (EpochNo 10)
+  mRes <- evalEither res
+  H.assert $ isJust mRes
+
+  -- Step 2. Propose motion of no confidence. DRep and SPO voting thresholds must be met.
+
+  -- Create proposal to add a new member to the committee
+
+  proposalAnchorFile <- H.note $ work </> "sample-proposal-anchor"
+  H.writeFile proposalAnchorFile "dummy anchor data"
+
+  proposalAnchorDataHash <- H.execCli' execConfig
+    [ eraToString era, "governance"
+    , "hash", "anchor-data", "--file-text", proposalAnchorFile
+    ]
+
+
+  proposalFile <- H.note $ work </> "sample-proposal-anchor"
+  stakeVkeyFp <- H.note $ work </> "stake.vkey"
+  stakeSKeyFp <- H.note $ work </> "stake.skey"
+
+  _ <- P.cliStakeAddressKeyGen tempAbsPath'
+         $ P.KeyNames { P.verificationKeyFile = stakeVkeyFp
+                      , P.signingKeyFile = stakeSKeyFp
+                      }
+  void $ H.execCli' execConfig $
+    [ eraToString era, "governance", "action", "create-no-confidence"
+    , "--testnet"
+    , "--governance-action-deposit", show @Integer 1_000_000 --- TODO: Get from protocol parameters
+    , "--deposit-return-stake-verification-key-file", stakeVkeyFp
+    , "--anchor-url", "https://tinyurl.com/3wrwb2as"
+    , "--anchor-data-hash", proposalAnchorDataHash
+    , "--out-file", proposalFile
+    ]
+
+  txbodyFp <- H.note $ work </> "tx.body"
+  epochStateView <- getEpochStateView (File configurationFile) (File socketPath)
```

Comment:
`getEpochStateView` is called earlier already

Reply:
Yep, I force pushed and fixed this already.